style.css

```diff
@@ -0,0 +1,1000 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style-type: none !important;
```

Comment:
- [ ] Please remove this `!important`, it's not a good practice.

Reply:
Thank you for your feedback, we will improve it as soon as possible.